pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -809,8 +836,9 @@ private String formatParticipantToStructuredJSON(
                         questionsAnswers
                 );
 
+                // We're only exposing a few substitutions as "attributes" on the activity instances.
                 Map<String, String> subs = participant.getActivityInstanceSubstitutions(instance.getId());
-                for (var name : exposedSubs) {
+                for (var name : ActivityAttributesCollector.EXPOSED_ATTRIBUTES) {
```

Comment:
Type instead of `var` would be informative here.

Reply:
Done